browser/aframeComponents/wearable-skin.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Component for selecting your character's skin in game.
+ */
+
+import AFRAME from 'aframe';
+const THREE = window.THREE;
```

Comment:
You shouldn't need to do this--THREE is a global variable everywhere in A-Frame, even if the linter doesn't think so. (To fix the linter not thinking so, you can just do `/* global THREE */` on the first line.)